README.md

```diff
@@ -36,11 +36,13 @@ jobs:
           github_token: ${{ secrets.github_token }}
 ```
 
+> Note: You need to use your [Personal Access Token](https://github.com/settings/tokens) if this workflow will be triggered by another one.
```

Comment:
1. How about s/another one/another workflow/ to clarify the meaning?
2. Can you add a link to the forum (or describe the problem more detail) and clearly state that using personal access token is workaround?

Reply:
I'm guessing #5 will be another solution. Users can create tag and bump semver at the same workflow by #5. 

This solution makes more sense instead of using personal access token.

Reply:
That is right. I think we can close that one.